src/ScaffoldInstallerPlugin.php

```diff
@@ -83,8 +83,11 @@ public function onPostInstallCmd(Event $event)
         $this->installGitignore();
         $this->installDdevCommand();
         $this->installHostingProviderSupport();
-        $this->installCICommands();
+        $this->installCICommands($event->getComposer());
         $this->installEnvSupport();
+        if ($this->hasPantheonConfigurationFiles()) {
+            $this->pantheonSystemDrupalIntegrationsWarning();
+        }
```

Comment:
Do we need changes to installCICommands() if we're also calling the check here? It seems like we're doing double work somehow.